site/content/en/docs/Integration Patterns/player-capacity.md

```diff
@@ -10,7 +10,6 @@ description: >
 {{< alpha
 title="Player Tracking and Allocation Player Filter"
 gate="PlayerTracking,PlayerAllocationFilter" >}}
-{{< beta title="Allocation State Filter" gate="StateAllocationFilter" >}}
```

Comment:
Same as above - need to wrap this in a feature shortcode to expire it on the next version, so it doesn't get removed immediately (docs get published immediately on merge to `main`).